courses/machine_learning/images/mnist_linear.ipynb

```diff
@@ -9,7 +9,12 @@
    "source": [
```

Comment:
<ul><li>grep tensorflow==2 -&gt; grep tensorflow==2.0.0</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Chatted with Benoit offline. Keeping libraries more general for easier maintenance.